syntax = "proto3";

message Imu {
    float x = 1;
    float y = 2;
    float z = 3;
    float roll = 4;
    float pitch = 5;
    float yaw = 6;
    float acceleration_x = 7;
    float acceleration_y = 8;
    float acceleration_z = 9;
    float velocity_x = 10;
    float velocity_y = 11;
    float velocity_z = 12;
    int64 timestamp = 13;
    string imu_id = 14;
}
    